spec/system/jobseekers/jobseekers_can_manage_a_profile_spec.rb

```diff
@@ -380,30 +374,21 @@
         end
       end
 
+      #  I think we could combine this with the test/tests on line 338 or line 402 or both
       context "if the jobseeker has a previous job application" do
         let!(:previous_application) { create(:job_application, :status_submitted, jobseeker:, create_details: true) }
 
         before { visit jobseekers_profile_path }
 
-        it "prefills the form with the jobseeker's work history" do
+        it "prefills the form with the jobseeker's work history and qualifications" do
```

Comment:
Combined two tests into one as I think they can comfortably be tested together and it reduces some set up costs.